extension/webpack.config.js

```diff
@@ -37,5 +37,14 @@ module.exports = {
                 }
             ],
         }),
-    ]
+    ],
```

Comment:
merge conflict code, see my comments in the previous pull request

Reply:
Done